content/general/concepts/what-is-cloud-computing/what-is-cloud-computing.md

```diff
@@ -0,0 +1,58 @@
+
+## Types of Cloud
+
+### 1. Public cloud
+
+Public clouds are owned and operated by third-party cloud service providers that deliver their computing resources like servers and storage over the Internet. Some examples include Amazon Web Services (AWS), Google Cloud Platform (GCP), and Microsoft Azure.
+
+### 2. Private cloud
+
+Private cloud refers to cloud computing resources used exclusively by a single business or organization. A private cloud can be physically located on the company’s on-site data center. Some companies also pay third-party service providers to host their private cloud.
+
+### 3. Hybrid cloud
+
+Hybrid clouds are the combination of both public and private clouds that allows data and applications to be shared between them. By allowing data and applications to move between private and public clouds, it provides business greater flexibility, more deployment options and helps optimize existing infrastructure, security and compliance.
+
+## Benefits of Cloud Computing
+
+### 1. Cost
+
+Cloud computing eliminates the capital expense of purchasing hardware and software, establishing and operating on-site datacenters, around-the-clock electricity for power and cooling, and IT experts that manage infrastructure. Users, on the other hand, only need to pay for the resources or services they use.
+
+### 2. Speed
+
+Most cloud providers provide on-demand services, so even large amounts of computing resources can be provisioned in minutes, typically with just a few mouse clicks, giving businesses a lot of flexibility and taking the pressure off capacity planning.
+
+### 3. Security
+
+Data security is a concern for every business, regardless of size and industry. Cloud Computing offers many advanced security features which guarantee that data is securely stored and handled. Cloud providers implement baseline protections for their platforms and the data they process such as authentication, access control, and encryption.
+
+### 4. Reliability
+
+Cloud computing makes data backup, disaster recovery, and business continuity simple and affordable because data can be mirrored at multiple redundant sites on the cloud provider’s network.
+
+### 5. Scalability
+
+Users do not have to over-provision resources for future peak levels of business activity with cloud computing. Instead, they only provide the resources that they require. Users can instantly grow and shrink capacity by scaling these resources up or down as your business needs change.
```

Comment:
In the second sentence, what does "they" refer to? Can we make it clear/explicit to avoid any possible confusion?